lib/smart_answer_flows/benefit-cap-calculator.rb

```diff
@@ -68,11 +68,8 @@ def define
         option :widow_pension
         option :widows_aged
 
-        next_node_calculation :benefit_related_questions do |response|
-          questions = response.split(",").map { |r| :"#{r}_amount?" }
-          questions << :housing_benefit_amount? if housing_benefit == 'yes'
-          questions << :single_couple_lone_parent?
-          questions
+        next_node_calculation :benefit_types do |response|
+          response.split(",").map(&:to_sym)
```

Comment:
Did you consider adding `:housing_benefit` to this `benefit_types` array? I _think_ that would mean you could simplify the current `else` in the `case` statements:

```ruby
# Old
case benefit_types.shift
...
else
  if housing_benefit == 'yes'
    question :housing_benefit_amount?
  else
    question :single_couple_lone_parent?
  end
end

# New
case benefit_types.shift
...
when :housing_benefit then :housing_benefit_amount?
else :single_couple_lone_parent?
end
```

Does that make sense, or am I missing something?